README.md

```diff
@@ -77,6 +77,13 @@ To run LIVE bundles on localhost:
 - Then run `rm -rf build && npm run build:live && npm run start`
 - Visit a live article: http://localhost:7080/news/articles/c8xxl4l3dzeo
 
+## Changing request location
+
+Some features perform differently dependant on whether a user is located within the UK or internationally. You can explicitly request a specific version by accessing Simorgh via a specific localhost BBC domain:
+
+- UK version: [http://localhost.bbc.co.uk:7080](http://localhost.bbc.co.uk:7080)
```

Comment:
Linking to article example would ensure the links would work directly, without needing to remember a correct url. 
```suggestion
- UK version: [http://localhost.bbc.co.uk:7080/news/articles/c0000000001o](http://localhost.bbc.co.uk:7080/news/articles/c0000000001o)
```